packages/common/http/src/params.ts

```diff
@@ -89,18 +89,28 @@ function paramParser(rawParams: string, codec: HttpParameterCodec): Map<string,
   }
   return map;
 }
+
+/**
+ * Encode input string with standard encodeURIComponent and then un-encode specific characters.
+ */
+const STANDARD_ENCODING_REGEX = /%(\d[a-z0-9])/gi;
```

Comment:
NIT I think this could be:

```suggestion
const STANDARD_ENCODING_REGEX = /%(\d[a-f0-9])/gi;
```

Reply:
Correct, just fixed, thank you!